### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ian-r-rose**
This is not perfect, but at least it looks a lot better on the github repo.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,84 +1,76 @@
...
-*** Requirements
+Contact: Ian Rose 
+ian.rose@berkeley.edu
 
-- Python 2.7 (not Python 3.x)
-- Python modules:
-  NumPy, SciPy, matplotlib
+## Requirements
 
+* Python 2.7 (not Python 3.x)
```
Comment (bobmyhill) [inline Readme.md]: Can this be changed to "Python 2.7 and later"? Are there any major problems with 3.x?

Comment (bobmyhill) [inline Readme.md]: (I mean ones which won't be fixed very shortly by your new PR?)

**Comment by tjhei** [comment]
looks good to me.

**Comment by bobmyhill** [comment]
Me too. Good to merge (my comment can be addressed with your next PR :) ).

File: Readme.md (hunk 1 above)
Comment (ian-r-rose) [inline Readme.md]: Yeah, I thought to change that, but decided to wait until #149 is merged.

**Pull request merged**